bindings/python/tests/tuple_tests.py

```diff
@@ -39,8 +39,8 @@
 def randomUnicode():
     while True:
         c = random.randint(0, 0xffff)
-        if unicodedata.category(unichr(c))[0] in 'LMNPSZ':
-            return unichr(c)
+        if unicodedata.category(chr(c))[0] in 'LMNPSZ':
```

Comment:
Does this work in Python 2? I'm not sure we've officially dropped support of it, so we might still be running these tests in Python 2.

Reply:
No, it's something different in Python 2. Yeah I guess it uses Python 2. A large part seems to be Python 2 looking at shebang lines and from documentation [here](https://github.com/apple/foundationdb/blob/main/documentation/sphinx/source/api-python.rst) it looks like FoundationDB is still supporting Python 2.

Are we using an actively maintained Python 2 interpreter for Python prod code? If not, it could be a security risk and we should migrate to Python 3.